doc/sections/ex-gwf-sfr-p01b.tex

```diff
@@ -0,0 +1,37 @@
+\section{Advanced Packages with MVR}
+
+% Describe source of problem
+The Advanced \mf Packages example problem is designed to demonstrate the combined use of the UZF, SFR and LAK Packages in \mf.  It originally appeared in the "Getting Started" pdf document in the \mf -docs.git repository, and is included here for completeness of the examples. 
```

Comment:
The quotes in this line are not ascii characters.  They should be removed or replaced with ascii quotes.